alpine/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM alpine:latest
+MAINTAINER Peter Martini <PeterCMartini@GMail.com>
```

Comment:
FYI, the `MAINTAINER` instruction is officially deprecated (see https://docs.docker.com/engine/reference/builder/#maintainer-deprecated).  For official images, we don't really mind if you remove it entirely, but you can also replace it with a `LABEL` if you prefer (as noted in that upstream document).